packages/dmn-js/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ All notable changes to [dmn-js](https://github.com/bpmn-io/dmn-js) are documente
 
 ___Note:__ Yet to be released changes appear here._
 
+## 17.1.0
+
+* `FIX`: formal parameters suggestions are available in real time ([#914](https://github.com/bpmn-io/dmn-js/pull/914))
```

Comment:
Given the only entry is a `fix`, and the closed issue mentioned a bug, we should release a patch release.
Let's also link to the dmn-js issue instead of pull request if possible: https://github.com/bpmn-io/dmn-js/issues/896
The changelog entries follow git convention of using imperative form, so this should be something like "display updated formal parameters in suggestions"

Reply:
Cf. https://semver.org/

Reply:
Updated.